src/main/kotlin/gg/essential/universal/wrappers/message/UMessage.kt

```
@@ -121,3 +121,21 @@ class UMessage {
         messageParts.forEach { _chatMessage.appendSibling(it) }
     }
 }
+
+private val printChatMessageWithOptionalDeletion: MethodHandle? = try {
+    val method = GuiNewChat::class.java.declaredMethods.find { method ->
+        method.parameterTypes.run {
+            size == 2 &&
+                    get(0) == MCITextComponent::class.java &&
+                    get(1) == Int::class.java
+        }
+    } ?: throw NoSuchMethodException("Couldn't find the right method for UMessage")
+    method.isAccessible = true
+    MethodHandles.lookup().unreflect(method)
+} catch (e: Throwable) {
+    null
```

Should definitely print the error, never just silently drop it.